label_studio/tasks/models.py

```diff
@@ -178,6 +179,20 @@ class Meta:
     def file_upload_name(self):
         return os.path.basename(self.file_upload.file.name)
 
+    @classmethod
+    def get_random(cls, project):
+        """Get random task from a project, this should not be used lightly as its expensive method to run"""
+        if project is None:
+            raise Exception('Project id is required')
```

Comment:
The method shouldn't raise an exception for this. `project` should be a required argument, and there shouldn't be any nullchecking.